ecs-agent/acs/session/task_manifest_common.go

```diff
@@ -13,10 +13,21 @@
 
 package session
 
+import "github.com/aws/amazon-ecs-agent/ecs-agent/acs/model/ecsacs"
+
 // ManifestMessageIDAccessor stores the latest message ID that corresponds to the
 // most recently processed task manifest message and is used to determine the
 // validity of a task stop verification ack message.
 type ManifestMessageIDAccessor interface {
 	GetMessageID() string
 	SetMessageID(messageID string) error
 }
+
+// SequenceNumberAccessor is used to get and set state for the current container
+// instance. It sets and gets the latest sequence number that corresponds to the most
+// recent task manifest that the task manifest responder has processed it alo sets the
+// latest message id
+type SequenceNumberAccessor interface {
```

Comment:
Could we move this interface to `ecs-agent/acs/session/task_manifest_responder.go` adjacent to where the `TaskComparer` interface is defined right now? My current understanding is that this interface is only needed for task manifest responder and is not common to both the task manifest responder and task stop verification ACK responder.